### PR TITLE
LTD-3693-LTD-3719-LTD-3720-LTD-3721-LTD-3723-Bugs

### PR DESCRIPTION
**Pull request by seijihg**
There is an API change related to LTD-3693
https://github.com/uktrade/lite-api/pull/1461

https://uktrade.atlassian.net/browse/LTD-3693
https://uktrade.atlassian.net/browse/LTD-3719
https://uktrade.atlassian.net/browse/LTD-3720
https://uktrade.atlassian.net/browse/LTD-3721
https://uktrade.atlassian.net/browse/LTD-3723

### CODE REVIEW

**Review by depsiatwal — APPROVED** [review]
lgtm

File: caseworker/assets/styles/_mentions.scss [hunk 1]
```diff
@@ -2,5 +2,14 @@
   &__new,
   &__urgent {
     margin-left: 0px;
+    padding: 5px 8px 4px 8px;
+
+    &--left-margin {
```
Comment (kevincarrogan) [inline caseworker/assets/styles/_mentions.scss:7]: Class name shouldn't describe the styling that gets attached and this is a good use of the modifier.

A class name should describe the thing.

Comment (kevincarrogan) [inline caseworker/assets/styles/_mentions.scss:7]: Class name shouldn't describe the styling that gets attached and this isn't what a modifier is for in BEM.

A class name should describe the thing.

File: caseworker/templates/users/mentions.html [hunk 2]
```diff
@@ -33,18 +33,18 @@ <h1 class="govuk-heading-l">
 						<td class="govuk-table__header" scope="row">
 								{{ user_case_note_mention.reference_code }}
 						</td>
-						<td class="govuk-table__cell"> {% if user_case_note_mention.is_accessed %}<p class="mentions__new govuk-tag">NEW</p>{% endif %} <br>{% if user_case_note_mention.is_urgent %} <p class="warning-text mentions__urgent">URGENT</p>{% endif %} </td>
+						<td class="govuk-table__cell"> {% if user_case_note_mention.is_accessed %}<p class="govuk-tag mentions__new">NEW</p>{% endif %} <br>{% if user_case_note_mention.is_urgent %} <p class="warning-text mentions__urgent">URGENT</p>{% endif %} </td>
```
Comment (kevincarrogan) [inline caseworker/templates/users/mentions.html:36]: Why is one of these things `govuk-tag` and the other `warning-text`?

File: caseworker/activities/templates/activities/notes-and-timeline.html [hunk 3]
```diff
@@ -79,9 +79,10 @@ <h2 class="notes-and-timeline-timeline__day-group-heading">{{ activities_by_date
                                                 {% if user.team %}<span class="app-activity__item__user">{{ user.team }}:</span> {% endif %}{{ user.first_name }} {{ user.last_name }}
                                             {% endif %}
                                             {% if activity.verb == "created_case_note_with_mentions" %}
-                                                {{ activity.text|slice:":-1"|linebreaks }}
+                                                {{ activity.text|slice:":-1"}}
                                                 {{ activity.payload.mention_users|join:", " }}
-                                                {% if activity.payload.is_urgent %} <p class="warning-text mentions__urgent">URGENT</p> {% endif %}
+                                                in a case note
+                                                {% if activity.payload.is_urgent %} <p class="warning-text mentions__urgent mentions__urgent--left-margin">URGENT</p> {% endif %}
```
Comment (kevincarrogan) [inline caseworker/activities/templates/activities/notes-and-timeline.html:85]: `mentions__urgent` is a confusing use of BEM here.

File: caseworker/templates/users/mentions.html (hunk 2 above)
Comment (seijihg) [reply caseworker/templates/users/mentions.html:36]: govuk-tag was used in other places thats why. and the warning-text hmmm it was there before :P

**Pull request merged**